core/src/inputs/mod.rs

```diff
@@ -31,6 +33,7 @@ pub use advice::AdviceSet;
 pub struct ProgramInputs {
     stack_init: Vec<Felt>,
     advice_tape: Vec<Felt>,
+    advice_tapes: TapesMap,
```

Comment:
Let's change `advice_tapes` to `advice_map`. Also, I don't think it's worth adding a type alias for `BTreeMap<[u8; 32], Vec<Felt>>`.